secret/cipher/cipher.go

```diff
@@ -0,0 +1,56 @@
+package cipher
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/md5"
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"io"
+)
+
+// Encrypt it encrypt's the key
+func Encrypt(key, plaintext string) (string, error) {
+	block, _ := newCipher(key)
+
+	ciphertext := make([]byte, aes.BlockSize+len(plaintext))
+	initnvector := ciphertext[:aes.BlockSize]
+	_, err := io.ReadFull(rand.Reader, initnvector)
+	if err != nil {
+		fmt.Println(err)
+	} else {
+		stream := cipher.NewCFBEncrypter(block, initnvector)
+		stream.XORKeyStream(ciphertext[aes.BlockSize:], []byte(plaintext))
+	}
+	return fmt.Sprintf("%x", ciphertext), err
+}
+
+// Decrypt it decrypt's the key value
+func Decrypt(key, cipherHex string) (string, error) {
+	var ciphertext []byte
+	block, err := newCipher(key)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
we can return from here, when ever you got error return immediately.

Reply:
and you can follow this all rest of the places

Reply:
Addressed the comments